test/parallel/test-child-process-silent.js

```diff
@@ -97,11 +97,11 @@ if (process.argv[2] === 'pipe') {
     parent.kill();
 
     // Check std(out|err) pipes
-    assert.ok(!stdoutData, 'The stdout socket was piped to parent');
-    assert.ok(!stderrData, 'The stderr socket was piped to parent');
+    assert.ok(!stdoutData);
+    assert.ok(!stderrData);
 
     // Check message system
-    assert.ok(childSending, 'The child was able to send a message');
-    assert.ok(childReciveing, 'The child was able to receive a message');
+    assert.ok(childSending);
+    assert.ok(childReceiving);
```

Comment:
Instead of just removing this it would probably be best to either change it to `assert.strictEqual` or to include the value itself in the error message. But now it would not be known what the value is.